evm/src/cpu/kernel/asm/rlp/num_bytes.asm

```diff
@@ -5,8 +5,12 @@ global num_bytes:
     DUP1 ISZERO %jumpi(return_1)
     // Non-deterministically guess the number of bits
     PROVER_INPUT(num_bits)
-    %stack (num_bits, x) -> (num_bits, x, num_bits)
-    %decrement
+    // stack: num_bits, x
+    SWAP1
+    PUSH 1
+    DUP3
+    // stack: num_bits, 1, x, num_bits
```

Comment:
nit: This is fine, but why not use a `%stack` macro here? I feel like it should be preferred for larger sequences of stack manipulation.

Reply:
Can do indeed :)